src/devTools.js

```diff
@@ -138,6 +143,7 @@ function liftReducer(reducer, initialState) {
         currentStateIndex++;
       }
       stagedActions = [...stagedActions, liftedAction.action];
+      timestamps = [...timestamps, Date.now()];
       break;
```

Comment:
I still think it's better to put timestamp into liftedAction and grab it from there. This way reducer is completely deterministic.